docs/_posts/GitTutorial-brief.md

```diff
@@ -0,0 +1,256 @@
+# Git this
+
+## What happens when you don't use version control?
+
+### A general life example:
+![](fig/phd_comics_VC.png)
+
+You most likely started out doing something like this. Maybe you've become more sophisticated (or not) and now you
+
+* date files
+* append `_vXXX`
+
+This is good, but you can still do better. 
+
+## Why version control is amazing
+* Code that works will be saved permanently
+* If you break the code that works, reverting is easy
+* You still only need to keep one version around (the VC program does the rest)
+* Collaboration is kept smooth and coordinated
+* Productivity is not stifled by too many people working on one document
+
+![](fig/rock-climbing.jpg)
+
+## Git
+
+[Git](https://git-scm.com/) is the method of version control we're going to be working with. Other methods exist (SVN and Mercurial are the big names, and Wikipedia's [page history](https://en.wikipedia.org/w/index.php?title=List_of_version_control_software&action=history) is a more commonly known example).
+
+Git tracks your files by essentially taking a snapshot of a directory or subdirectory structure and saving it over time. The process of capturing one of these processes is called a commit.
+
+To be a little more specific, you can think of Git as working in three different areas or bins. There is (1) the workspace, (2) the repository, and (3) the index. 
+
+#### Workspace: 
+The set of directories and files that you actually operate on.
+
+#### Repository (repo)
+The set of linked commits corresponding to snapshots of the workspace at specified points in time.
+
+#### Index
+The staging area where you "set up your snapshot"; files in the index that have changed since your last commit will be updated on your next commit.
+
+### Hashes
+Each commit is given a unique label (called a hash) and is tied to the previous commit. This is created using a function which converts a set of information (i.e. your docs) into a string of letters and numbers. Hashes include a layer of security, since each relies on its parent's hash.
+
+### Diagramatic Representation
+Below is a diagram of the git process for a single file; I've named it file_1. You start by creating the file in your workspace. 
+
+![](fig/git_wd01.png)
+
+Once you are satisfied with it's progress, you decide that you want to commit your work. You add the file to the index. 
+
+![](fig/git_index01.png)
+
+Now you're ready. You can commit the file to your repository. (Notice that in the repository file_1 is shown with the hash of the commit. In reality the full hash for a git commit is much longer.)
+
+![](fig/git_repo01.png)
+
+Now, if you make some changes to file_1. Your workspace changes. 
+
+![](fig/git_wd02.png)
```

Comment:
This figure looked the same as the previous one -- perhaps add a red mark to the 'file' icon? Same applies to the figures below.